src/main/features/dashboard/views/macro/caseHistory.njk

```diff
@@ -85,6 +87,9 @@ ccjRequestedForDefendantClaimDetails %}
           {{ caseStatusForClaimant(state.status, claim) }}
         </p>
       {% endif %}
+      {% if state.status === ClaimStatus.PAID_IN_FULL_CCJ_CANCELLED or PAID_IN_FULL_CCJ_SATISFIED %}
```

Comment:
This is asking if (state is paid in full ccj cancelled) or (the constant PAID_IN_FULL_CCJ_SATISFIED is defined)

Reply:
Good spot! Didn't realise I needed `state.status ===` in front of `PAID_IN_FULL_CCJ_SATISFIED` after the `or` thanks